packages/flutter_tools/lib/src/base/process_manager.dart

```diff
@@ -491,8 +491,8 @@ class _RecordingProcess implements Process {
   @override
   Future<int> get exitCode => delegate.exitCode;
 
-  @override
-  set exitCode(Future<int> exitCode) => delegate.exitCode = exitCode;
+  // ignore: annotate_overridden_members
+  set exitCode(Future<int> exitCode) => throw new UnsupportedError('set exitCode');
```

Comment:
Maybe add a TODO about removing this once we get updated version of the std library in both target and host?